AsyncDisplayKit/ASButtonNode.h

```diff
@@ -36,11 +37,66 @@
  */
 @property (nonatomic, assign) ASVerticalAlignment contentVerticalAlignment;
 
+/**
+ *  Returns the styled title associated with the specified state.
+ *
+ *  @param state The state that uses the styled title. The possible values are described in ASControlState.
+ *
+ *  @return The title for the specified state.
+ */
+- (NSAttributedString * _Nullable)attributedTitleForState:(ASControlState)state;
 
-- (NSAttributedString *)attributedTitleForState:(ASControlState)state;
-- (void)setAttributedTitle:(NSAttributedString *)title forState:(ASControlState)state;
+/**
+ *  Sets the styled title to use for the specified state.
+ *
+ *  @param title The styled text string to use for the title.
+ *  @param state The state that uses the specified title. The possible values are described in ASControlState.
+ */
+- (void)setAttributedTitle:(nullable NSAttributedString *)title forState:(ASControlState)state;
```

Comment:
We need to clearly describe on this as well as the title + color APIs that the two are mutually exclusive / overwrite each other, right?